readthedocs/projects/tests/test_build_tasks.py

```diff
@@ -197,6 +197,11 @@ def test_build_updates_documentation_type(self, load_yaml_config):
             }
         )
 
+        # Create the artifact paths, so it's detected by the builder
```

Comment:
Could use a bit more context here. I think you're trying to say:

```suggestion
        # Create the artifact paths, so that our mocked objects
        # get picked up as having been "built".
```

Reply:
Hrm, it's not for the mocked objects. We are doing checks for path existence to know if a format was built in the build or not to decide if it has to be uploaded to the storage. This is done at https://github.com/readthedocs/readthedocs.org/blob/faa611fad689675f81101ea643770a6b669bf529/readthedocs/projects/tasks/builds.py#L798-L804

I'm not sure how to make it clearer. I'd appreciate if you have a suggestion to expand that comment now you have a little more context about the logic behind it.

Reply:
Updated the comment.